From 839f12a45ea41e7a2d931bd580594fc76f878ee6 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Mon, 9 Sep 2024 19:03:36 -0400
Subject: [PATCH] Include map header in hud_elements.h

Otherwise, I get the following compilation errors on OSX

```
2024-09-09T22:58:12.8720860Z In file included from ../src/vulkan.cpp:49:
2024-09-09T22:58:12.8751290Z In file included from ../src/overlay.h:12:
2024-09-09T22:58:12.8785670Z ../src/hud_elements.h:147:14: error: no template named 'map' in namespace 'std'; did you mean 'max'?
2024-09-09T22:58:12.8805510Z   147 |         std::map<VkPresentModeKHR, std::string> presentModeMap = {
2024-09-09T22:58:12.8821870Z       |         ~~~~~^~~
2024-09-09T22:58:12.8838900Z       |              max
```

full log to be attached soo
---
 src/hud_elements.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/hud_elements.h b/src/hud_elements.h
index 90bd25f17e..1b6622aded 100644
--- a/src/hud_elements.h
+++ b/src/hud_elements.h
@@ -2,6 +2,7 @@
 #include <vector>
 #include <string>
 #include <utility>
+#include <map>
 #include <imgui.h>
 #include "timing.hpp"
 #include <functional>
